index.js

```diff
@@ -1437,6 +1437,22 @@
     }
   };
 
+  //# not :: Boolean -> Boolean
+  //.
+  //. Takes a Boolean and returns the negation of that value
+  //. (`false` for `true`; `true` for `false`).
+  //.
+  //. ```javascript
+  //. > S.not(true)
+  //. false
+  //.
+  //. > S.not(false)
+  //. true
+  //. ```
+  S.not = def('not', [Boolean], function(x) {
+    return !x.valueOf();
+  });
```

Comment:
Without `valueOf` the function will fail to evaluate correctly when passed a Boolean created with `new` in this case:

```js
S.not(new Boolean(false)) // returns false instead of true
```
I guess it's because objects are truthy so !(new Boolean(false)) will always evaluate to `false`.